rdkit/Chem/PandasPatcher.py

```diff
@@ -0,0 +1,235 @@
+from io import StringIO
```

Comment:
I'm not going to pretend that I read through this in any detail at all

Reply:
Still, thanks for pulling it out of PandasTools.py, which is now more approachable

Reply:
Going through all versions of `pandas` from 0.25.3 onwards to make sure the patching works on all of them was not particularly entertaining, but hopefully we have something more robust in place now.